sumologic/async.py

```diff
@@ -0,0 +1,162 @@
+from copy import copy
```

Comment:
I don't like it that the whole file is basically copied, but I couldn't find a better way. Is there?